csharp/ql/src/Security Features/CWE-502/UnsafeDeserialization.ql

```diff
@@ -5,16 +5,13 @@
  * @kind problem
  * @id cs/unsafe-deserialization
  * @problem.severity warning
+ * @precision low
```

Comment:
Where did this value come from? I think medium/warning might be sufficient in order to get the query run but excluded.

Reply:
I am not sure we even want to run it; It always produces a superset of the results produced by the more ambitious `UnsafeDeserializationUntrustedInput.ql` query.